.github/workflows/pr.yaml

```diff
@@ -0,0 +1,117 @@
+name: PR checks
+on: pull_request
+env:
+  SECRET_KEY: insecure_test_key
+
+jobs:
+  run-image:
+    runs-on: ubuntu-latest
```

Comment:
I'd prefer if we pinned this to the version we're actually deploying to...this change will need to be made all the way down

Reply:
@carkod the version we are deploying is not `ubuntu-18.04` and according the docs `ubuntu-18.04` is currently `ubuntu-latest`. So this is effectively not fixed. If you really want to pin this `ubuntu-20.04` is what you are looking for. https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

Reply:
The problem I found is that `ubuntu-20.04` is not really the official one, as @jpmartinspt points in that link it's a preview, and it is giving me problems because it doesn't have a bunch of packages installed such as docker.

Reply:
I was mostly pointing out that `18.04` is the same as `latest` in github actions context. In any case our multistage builds use images like `node:12-slim` to build `css` and `js`  and that is based of debian, so setting an action to run on a particular verision of ubuntu does not guarantee much. We are probably more interested in making sure we use the right node and python versions(looking at the dockerfile of each project) in some of the steps(lints and tests of python and js code). See https://help.github.com/en/actions/language-and-framework-guides/using-nodejs-with-github-actions and https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions. Bring this up tomorrow in stand-up, it's worth discussing and possibly revisiting in repos that are using actions already.

Reply:
Our current CircleCI workflows do not test anything against a production like image, when we lint or run your tests with `./run` or `dotrun` locally nothing is run in a production like container and our current demos don't use the same images that go into staging and production . We can build the production image in CI and we are introducing a step that does it to make sure nothing in a PR breaks the image building. The new demoservice containers will be built from the same Dockerfile we use for staging and production and both these things are valuable. For things like prettier, flake, black and python unittests, I believe picking the matching node/python versions is probably enough and that what I suggested in the comment above.

Reply:
Well I just think that we don't want to be updating this every time there is a new version of ubuntu. Furthermore, I don't think that fixing `runs-on` matters too much, as this is what Github actions use to set up the workflow, we are still testing the building and running of the image in the following instructions, using our `Dockerfile` which uses focal.